build/PackageDiffIgnore.xml

```diff
@@ -655,6 +655,9 @@
 				<Member
 					fullName="System.Void Windows.ApplicationModel.DataTransfer.DataPackage.SetText(System.String text)"
 					reason="UWP uses 'value' as the parameter name" />
+				<Member
```

Comment:
This method should most likely be internal. The ignore will be useful then :)

Reply:
Changed to internal

Reply:
@jeromelaban Oh, switching to internal means we would need to add all the platform versions into diff ignore - for example `System.Void Windows.UI.Xaml.Controls.IconElement.AddIconElementView(Android.Views.View child)`, etc., is that ok?

Reply:
@jeromelaban I have added the platform variants in the diff ignore, so now the build should go through, also added mention of `AddIconElementView` in breaking changes, can I add ready to merge?